CMakeLists.txt

```diff
@@ -10,7 +10,10 @@ include_directories(${CPP_STATSD_CLIENT_DIR})
 install(DIRECTORY ${CPP_STATSD_CLIENT_DIR} DESTINATION include)
 
 find_package(Threads)
-add_executable(testStatsdClient ${CMAKE_CURRENT_SOURCE_DIR}/tests/testStatsdClient.cpp)
+add_executable(testStatsdClient 
+  ${CMAKE_CURRENT_SOURCE_DIR}/tests/testStatsdClient.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/tests/StatsdServer.hpp)
+target_include_directories(testStatsdClient PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/tests)
```

Comment:
making sure the test can actually see and use the mock statsd server implementation.